src/js/content_scripts/input-mapper-utils.js

```diff
@@ -188,4 +188,156 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
             }, that.options.frequency);
         }
     };
+
+    /**
+     *
+     * Tab through the webpage using analog sticks.
+     *
+     * @param {Object} that - The inputMapper component.
+     * @param {Integer} value - The value of the gamepad input.
+     * @param {Integer} speedFactor - Times by which the tabbing speed should be increased.
+     * @param {Boolean} invert - Determine if the tabbing should be in opposite order.
+     *
+     */
+    gamepad.inputMapperUtils.analogTabbing = function (that, value, speedFactor, invert) {
+        var inversionFactor = invert ? -1 : 1;
+        value = value * inversionFactor;
+        if (value > 0.20) {
```

Comment:
As with other cuttoff values, the bounds should be configurable.

Reply:
I'm sorry. That was a silly mistake on my part. It should have been 0 instead as the case with joysticks on the gamepad. I'll rectify this.